internal.go

```diff
@@ -28,7 +28,10 @@ func makeGenesisDAG(daemon *docker.Client) error {
 	log15.Debug("created ethash container")
 	defer func() {
 		log15.Debug("deleting ethash container")
-		daemon.RemoveContainer(docker.RemoveContainerOptions{ID: ethash.ID, Force: true})
+		err := daemon.RemoveContainer(docker.RemoveContainerOptions{ID: ethash.ID, Force: true})
+		if err != nil {
+			log15.Error("failed to delete ethash container ", "error", err)
```

Comment:
I guess that's fine too. I figured it might be useful to see the container id but it's not that relevant. Please add it to the tester though since there might be multiple testers per hive run.